src/ecdsa/test_der.py

```diff
@@ -396,6 +398,128 @@ def test_with_malformed_tag(self):
         self.assertIn("constructed tag", str(e.exception))
 
 
+class TestRemoveImplicit(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.exp_tag = 6
+        cls.exp_data = b"\x0a\x0b"
+        # data with application tag class
+        cls.data_application = b"\x46\x02\x0a\x0b"
+        # data with context-specific tag class
+        cls.data_context_specific = b"\x86\x02\x0a\x0b"
+        # data with private tag class
+        cls.data_private = b"\xc6\x02\x0a\x0b"
```

Comment:
We probably should have a test for length > 128 to make sure the llen encoding/decoding is happening correctly

Reply:
this is reusing common code for encoding and decoding length, so I'm quite sure that it will handle it correctly